srcpkgs/tzupdate/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'tzupdate'
 pkgname=tzupdate
-version=2.0.0
-revision=3
+version=2.1.0
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-setuptools python3-requests"
```

Comment:
```suggestion
depends="python3"
```

Reply:
Have you looked at their code or is there another trick to easily find out dependencies easily?